.github/CODEOWNERS

```
@@ -1,3 +1,20 @@
+# Fallback owners for code review - ensure all PRs have someone assigned for review.
+# (the last match will used so this is only used if there is no more specific code owner below)
+
+# Backend
+# is the first and gets everything to make things easier from matching syntax
+*                                             @nextcloud/server-backend
+
+# Frontend
+# this will override the backend code owners if needed
+__mocks__                                     @nextcloud/server-frontend
+__tests__                                     @nextcloud/server-frontend
```

I think we only have those in the root so maybe better

```suggestion
/__mocks__                                    @nextcloud/server-frontend
/__tests__                                    @nextcloud/server-frontend
```